csrc/scheduler/matmul_hopper+.cpp

```diff
@@ -566,17 +584,81 @@ void HopperPlus::scheduleMmaResults() {
     }
 
     transformLikeMmaOutputWithK(mma_result);
+    setMmaResultAllocationDomain(mma_result);
 
-    auto s = mma_utils::MmaSwizzler::scheduleMmaOutputAllocation(
-        mma_result->getLoopDomain());
-    mma_result->setAllocationDomain(s.as<IterDomain*>(), true);
     mma_result->axis(-1)->parallelize(ParallelType::Mma);
     mma_result->axis(-2)->parallelize(ParallelType::Mma);
     mma_result->axis(-3)->parallelize(ParallelType::Mma);
   }
 }
 
-void HopperPlus::scheduleEpilogueWithoutSmemEpilogue() {
+void HopperPlus::scheduleEpilogueWithoutSmemEpilogueBlackwell() {
+  std::vector<TensorView*> cached_tvs;
+  std::vector<TensorView*> propagate_to =
+      splitk_sums_.empty() ? mma_results_ : splitk_sums_;
+  std::vector<TensorView*> tmem_ld_tvs;
+  for (auto mma_result : mma_results_) {
+    TensorView* tmem_ld_tv = cacheAfter(mma_result);
+    tmem_ld_tv->definition()->as<LoadStoreOp>()->setOpType(
+        LoadStoreOpType::LdTMem);
+    tmem_ld_tvs.push_back(tmem_ld_tv);
+  }
+  for (auto& [c, c_cache] : cached_epilogue_inputs_) {
+    cached_tvs.push_back(c_cache);
+    propagate_to.push_back(c);
+  }
+  for (Val* dv : fusion_->outputs()) {
+    TensorView* d = dv->as<TensorView>();
+    NVF_ERROR(d->definition() && d->definition()->isA<LoadStoreOp>());
+
+    // Apply the default scheduling that is common to all register
+    // TensorViews after wgmma.
+    blockTileTensors({d});
+    parallelizeBlocks({d});
+    transformLikeMmaOutputWithoutK(d);
+
+    d->axis(-2)->parallelize(ParallelType::TIDx);
+    int64_t tmem_vectorize_factor = 1;
+    const int64_t n_mma = getN(params_->mma_macro);
+    while (n_mma % tmem_vectorize_factor == 0 && tmem_vectorize_factor <= 128) {
+      tmem_vectorize_factor *= 2;
+    }
+    if (tmem_vectorize_factor < n_mma) {
+      d->split(-1, tmem_vectorize_factor);
```

Comment:
Documentation ideas: 
`// After tmem_vectorize_factor: [..., Mo * No, Mw, Nw, Mi, Ni / tmem_vectorize_factor, tmem_vectorize_factor]`

Reply:
Added a comment in code:

```C++
    // TIDx is 128, so we use it for lanes of the accumulator. Also, we
    // vectorize the TMem load with a factor of v (tmem_vectorize_factor).
    // [..., Mo * No, Mw, Nw, Mi (TIDx), Ni / v, v (Vectorize)]
```